docs/guides/formulasC2/examples.md

```diff
@@ -0,0 +1,275 @@
+  5. Click **Variables**.
+  ![Variables](img/variables.png)
+  7. Click **Value**.
+  8. Enter a name for variable that represents the resource that you want to sync. For this tutorial we'll call it `resourceName`.
+  9. Click **Save**.
+  11. Create Element Instance variables to represent the source and target systems to sync. For this example, use `originInstance` and `destinationInstance`.
+10. In the formula visualization, click <img src="img/btn-add-step.gif" alt="Add a Step" class="inlineImage"> to add a step.
+6. Create a JS Filter step that makes sure that the bulk query is completed.
+  7. Click **JS Filter (true/false)**.
+  8. Enter a name for the script. We'll call it `isSuccessful`.
+  8. Enter a script such as the example below.
+
+      ```js
+      let status = trigger.args.status;
+
+      if (status && status === "COMPLETED") {
+        done(true);
+      } else {
+        done(false);
+      }
+      ```
+6. Create a JS Script step that defines an identifier field, which is the unique key for an upsert operation. It also specifices the content type as **csv**. Under the **isSuccessful** step, click the **OnSuccess** button  <img src="img/btn-onSuccess.png" alt="OnSuccess" class="inlineImage">.
+  7. Click **JS Script**.
+  8. Enter a name for the script. We'll call it `buildMetaData`.
+  8. Enter a script like the following example:
+
+      ```js
+      let metaData = {
+        "identifierFieldName":"email"
+      }
+
+      let downloadHeaders = {
+        "Accept":"text/csv"
+      };
+
+      done({
+        "metaData": metaData,
+        "downloadHeaders": downloadHeaders
+      });
+       ```
+1. Create an Element Stream step to move the files downloaded from the origin instance to the destination instance<. Under the **buildMetaData** step, click the **OnSuccess** button  <img src="img/btn-onSuccess.png" alt="OnSuccess" class="inlineImage">.
+  7. Select **Stream File**.
+  8. Enter a name. For this example we'll call it `bulkStream`.
+  9. In **Download Element Instance Variable**, click <img src="img/btn-Add.png" alt="Alt Text" class="inlineImage">, and then select the **originInstance** variable that we created earlier.
+  9. In **Download Method**, enter `GET`.
+  10. In **Download API**, enter `/hubs/crm/bulk/${trigger.args.id}/${config.resourceName}`. `${trigger.args.id}` gets the id from the payload sent to the trigger by [Formula 1](#formula-1). `${config.resourceName}` refers to the resourceName variable that identifies the resource that you want to sync</span>.
+  9. In **Upload Element Instance Variable**, click <img src="img/btn-Add.png" alt="Alt Text" class="inlineImage">, and then select the **destinationInstance** variable that we created earlier.
+  9. In **Upload Method**, enter `POST`.
+  10. In **Upload API**, enter `/hubs/crm/bulk/${config.resourceName}`. `${trigger.args.id}`.
```

Comment:
Yes, it's describing the components of the Download API field. The full context is:
${trigger.args.id} gets the id from the payload sent to the trigger by Formula 1. ${config.resourceName} refers to the resourceName variable that identifies the resource that you want to sync.